upi/openstack/inventory.yaml

```diff
@@ -34,6 +34,7 @@ all:
       # 3 is the minimum number for a fully-functional cluster.
       os_compute_nodes_number: 3
       aci_cni:
+        snatIP: 10.0.0.0/21
```

Comment:
We have already defined a config for this, see "cluster_snat_policy_ip" further below

Reply:
Right, fixed!